### PR TITLE
List info API 86ep539mv

### PR DESCRIPTION
**Pull request by arieljovananda88**
API to fetch list infos based on search content, category, and has the info been read or not

Testing Evidence:
Screenshot 2024-04-29 at 02 05 47
Screenshot 2024-04-29 at 02 05 55
Screenshot 2024-04-29 at 02 06 06
Screenshot 2024-04-29 at 02 06 16
Screenshot 2024-04-29 at 02 06 25
Screenshot 2024-04-29 at 02 06 41
Screenshot 2024-04-29 at 02 06 49
Screenshot 2024-04-29 at 02 07 02
Screenshot 2024-04-29 at 02 10 47

### CODE REVIEW

File: src/types/info.types.ts [hunk 1]
```diff
@@ -36,3 +36,76 @@ export const InfoSchema = createSelectSchema(infos, {
...
+export const ListInfoParamsSchema = z.object({
+  search: z
+    .string()
+    .optional()
+    .openapi({
+      param: {
+        name: 'search',
+        in: 'path',
+      },
+      example: '',
```
Comment (fawwazabrials) [inline src/types/info.types.ts]: ini 'in'-nya harusnya di query bukan path

File: src/types/info.types.ts [hunk 2]
```diff
@@ -36,3 +36,76 @@ export const InfoSchema = createSelectSchema(infos, {
...
+    }),
+  category: z
+    .string()
+    .optional()
+    .openapi({
+      param: {
+        name: 'category',
+        in: 'path',
+      },
+      example: 'cat',
```
Comment (fawwazabrials) [inline src/types/info.types.ts]: ini kebawah2 juga ganti kayak atas

File: src/routes/info.route.ts [hunk 3]
```diff
@@ -64,3 +66,31 @@ export const createInfoRoute = createRoute({
...
 });
+
+export const listInfoRoute = createRoute({
+  operationId: 'listInfo',
+  tags: ['info'],
+  method: 'get',
+  path: '/info',
+  request: {
+    params: ListInfoParamsSchema,
+  },
```
Comment (fawwazabrials) [inline src/routes/info.route.ts]: request buat di query harusnya bukan gini, coba cek open-graph.route buat referensi

**Review by fawwazabrials — DISMISSED** [review]

**Review by arsaizdihar — APPROVED** [review]
LGTM

**Pull request merged**